store/types/store.go

```diff
@@ -164,6 +164,9 @@ type CommitMultiStore interface {
 	// Panics on a nil key.
 	GetCommitKVStore(key StoreKey) CommitKVStore
 
+	// Panics on a nil version.
+	GetCommitInfoFromDb(ver int64) (*CommitInfo, error)
```

Comment:
```suggestion
	GetCommitInfoFromDB(ver int64) (*CommitInfo, error)
```

Reply:
Will change this. Unrelated note:

Do you see any problem with me making this a public function since it was private before? Just double checking

Reply:
Nope!